ui/src/app/pages/servermapping/servermapping.component.ts

```diff
@@ -264,23 +268,31 @@ export class ServermappingComponent implements OnInit {
   }
   loading:boolean = false;
   emptyResult:boolean = false;
+  notEnoughOnepage:boolean = false;
   getAssets(){
     this.searchBtnDisabled = true;
     this.searchBtnState = true;
     this.loading = true;
-    if(!this.emptyResult){
       this.service.getAssets(this.pageSizeAsset,this.currentPageAsset,this.keywords,this.category).subscribe(data=>{
         if(data.status == 200){
           this.searchBtnDisabled = false;
           this.searchBtnState = false;
           this.loading = false;
-          if(data.json().content.length != 0){
-            this.assets = data.json().content;
+          this.assets = data.json().content;
+          this.assets.forEach(element => {
+            element.enable = true;
+          });
+          if(data.json().content.length == 5){
```

Comment:
why it's 5 , idealy it should be the pagesize by default.

Reply:
When the size of conten list equal 5, there may be more datas, Otherwise it's the last page.

Reply:
The page size will not be changed.